src/sponsors/models.py

```diff
@@ -29,6 +32,7 @@ class Sponsor(ConferenceRelated):
     logo_svg = models.FileField(
         verbose_name=_('logo (SVG)'),
         blank=True, upload_to=logo_upload_to,
+        storage= default_storage if settings.DEBUG else GoogleCloudStorage(),
```

Comment:
Adopting the `settings.DEBUG` as the switch to determine whether to use `GoogleCloudStorage` leads to the discrepancy of migration scripts in dev & production environments, which should cause pain when trying to deploy a new version.